payments/tests/test_reservation_api.py

```diff
@@ -650,9 +652,11 @@ def test_approve_paid_reservation(
 
     1. Reservation new state should be WAITING_FOR_PAYMENT
     2. Payment is initiated
-    3. (TODO) Payment link is emailed to customer
+    3. Payment link is emailed to customer
     """
 
+    settings.RESPA_MAILS_ENABLED = True
```

Comment:
Could also do this with the `override_settings` decorator from `django.test.utils`:

```
@override_settings(RESPA_MAILS_ENABLED=True)
def test_approve_paid_reservation(
...
```

No need to change it, though.